test/jasmine/tests/gl3d_plot_interact_test.js

```diff
@@ -394,7 +394,7 @@ describe('Test gl3d plots', function() {
 
     it('@gl should be able to reversibly change trace type', function(done) {
         var _mock = Lib.extendDeep({}, mock2);
-        var sceneLayout = { aspectratio: { x: 1, y: 1, z: 1 } };
+        var sceneLayout = { aspectratio: { x: 1, y: 1, z: 1 }, dragmode: 'turntable' };
```

Comment:
Why did you need to change this? I'm fearing a breaking change here.

Reply:
So without setting `dragmode: 'turntable'` here, it would default to `'orbit'`? 

What are the `camera` settings in this case?

Reply:
@etpinard notice this isn't a setting, it's an expectation. But I agree, we shouldn't be mutating the input `scene`, per my [comment above](https://github.com/plotly/plotly.js/pull/3256/files#r235165793)